src/global.d.ts

```diff
@@ -8,6 +8,7 @@ interface Ticket {
   base: TicketBase;
   url: TicketUrl; // url = base + number
   description?: TicketDescription;
+  aliases: Array<String>;
```

Comment:
I think we'll want  [`string` instead of `String`](https://stackoverflow.com/a/34993650) here -- everything else looks 💯